demo/site/src/middleware/preview.ts

```diff
@@ -6,7 +6,7 @@ import { type CustomMiddleware } from "./chain";
 export function withPreviewMiddleware(middleware: CustomMiddleware) {
     return async (request: NextRequest) => {
         // Block preview is always allowed
-        if (request.nextUrl.pathname.startsWith("/block-preview/")) {
+        if (request.nextUrl.pathname.startsWith("/block-preview/") || request.nextUrl.pathname === "/block-preview") {
```

Comment:
with the new cookie `startsWith("/block-preview/")` is not needed anymore I think